src/edu/ucsd/library/xdre/tab/TabularRecord.java

```diff
@@ -259,22 +261,15 @@ protected void addCommonDescriptiveData(Element e, String objectID, String key)
         }
 
         // date /////////////////////////////////////////////////////////////////////////
-        if (key.startsWith("date:")) {
-            String type = key.substring(key.indexOf(":") + 1);
-            String date = data.get(key);
-
-            if (key.equalsIgnoreCase("date:creation")) {
```

Comment:
@lsitu - How come we don't check for date:creation anymore?  Does that mean any date element will have 'begin date' and 'end date'?

Reply:
@VivianChu The `begin date` and `end date` can bind to any dates if there are more thab one day exists. And it doesn't matter what date it is if there are only one date and we just bind begin/end date to it.

Reply:
Thanks for the clarification.  Overall, the PR looks good to me 👍